WebApp-OpenIDConnect-DotNet/Views/Shared/_LoginPartial.cshtml

```diff
@@ -2,6 +2,8 @@
 {
     <ul class="nav navbar-nav navbar-right">
         <li class="navbar-text">Hello @User.Identity.Name!</li>
+        <li><a asp-area="" asp-controller="Session" asp-action="EditProfile">Edit Profile</a></li>
+        <li><a asp-area="" asp-controller="Session" asp-action="ResetPassword">Reset Password</a></li>
```

Comment:
It feels a bit weird to me to show separate links for resetting your password and editing your profile. On most websites I believe you would expect to be able to edit your profile and reset your password on the same page. Is there a way we could show a similar experience? Can you inline the B2C UI onto a user profile management page somehow?